pootle/apps/pootle_app/management/commands/contributors.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import os
+os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
+
+from collections import Counter
+from django.contrib.auth import get_user_model
+from pootle_app.management.commands import PootleCommand
+from pootle_store.models import Unit
+
+
+User = get_user_model()
+
+
+class Command(PootleCommand):
+    help = "Update database stores from files."
+
+    def handle_all(self, **options):
+        units = Unit.objects.values("submitted_by").exclude(submitted_by=None)
+
+        if self.projects:
+            units = units.filter(store__translation_project__project__code__in=self.projects)
+
+        if self.languages:
+            units = units.filter(store__translation_project__language__code__in=self.languages)
+
+        contribs = Counter()
+        for v in units:
+            contribs.update((v["submitted_by"], ))
+
+        self.list_contributions(contribs)
+
+    def list_contributions(self, contribs):
+        for id, count in contribs.items():
+            user = User.objects.get(id=id)
+            if user.is_system():
+                continue
```

Comment:
Perhaps it is better to exclude `system` user in the `units` queryset above instead of having this `if` here.